docusaurus/docs-internals/editor-controls/unit-controls/slider.md

```diff
@@ -0,0 +1,64 @@
+|      `config.min`       |    number     |    0    | The minimum value that can be obtained by dragging the slider handle                                                                                                                                                                                                            |
+|    `config.inputMin`    |    number     |    -    | The minimum value that can be entered directly into the input field. If the user inputs a value lower than this, it will be automatically corrected to this minimum value                                                                                                       |
+|      `config.max`       |    number     |   100   | The maximum value that can be obtained by dragging the slider handle                                                                                                                                                                                                            |
+|    `config.inputMax`    |    number     |    -    | The maximum value that can be entered directly into the input field. If the user inputs a value higher than this, it will be automatically corrected to this maximum value                                                                                                      |
+|      `config.step`      |    number     |    1    | The step interval for changing the slider value                                                                                                                                                                                                                                 |
+|     `config.units`      | `Array<Unit>` |   [ ]   | The units of measurement displayed next to the slider value. Accepts all possible CSS units ("px", "%", "em" and others) <br/> <b>`Unit: { value: string; title: string; }`</b>                                                                                                 |
+|      `config.size`      |    string     |    -    | The size of the slider component, can be "short", "medium", "large" or "auto"                                                                                                                                                                                                   |
+| `config.debounceUpdate` |    boolean    |    -    | Determines whether the UI is updated using debouncing instead of throttling                                                                                                                                                                                                     |
+|   `config.updateRate`   |    number     |   16    | The delay in milliseconds after which the UI responds to value changes                                                                                                                                                                                                          |
+
```

Comment:
add `helper.content` and `helper.position` to this table if the option can render the helper.

Example here: https://github.com/EasyBrizy/Brizy-Local-Editor/pull/214/files#diff-10490b6760b9fb61b068a294a82ccb06423c1adef46072a3cd086954adf6657cR24

Check also with Tudor to be the same text descriptions